composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "enm/json-api-client",
+  "name": "stixxx2k/json-api-client",
```

Comment:
If I resolve this manually, your package name in the master branch will change to mine. Is that ok for you? If not, you have to create a new PR based on a branch where this matches my composer package name.

Reply:
Misinterpreted the PR mechanics. I've reverted the modification.